src-docs/src/views/form_controls/form_controls_example.js

```diff
@@ -129,6 +131,75 @@ const formControlLayoutRangeSource = require('!!raw-loader!./form_control_layout
 
 export const FormControlsExample = {
   title: 'Form controls',
+  intro: (
+    <>
+      <EuiText>
+        <p>
+          EUI provides different types of basic form controls:{' '}
+          <Link to="#text-field">
+            <strong>EuiFieldText</strong>
+          </Link>
+          ,{' '}
+          <Link to="#search-field">
+            <strong>EuiFieldSearch</strong>
+          </Link>
+          ,{' '}
+          <Link to="#number-field">
+            <strong>EuiFieldNumber</strong>
+          </Link>
+          ,{' '}
+          <Link to="#password-field">
+            <strong>EuiFieldPassword</strong>
+          </Link>
+          ,{' '}
+          <Link to="#select">
+            <strong>EuiSelect</strong>
+          </Link>
+          ,{' '}
+          <Link to="#textarea">
+            <strong>EuiTextArea</strong>
+          </Link>
+          , and{' '}
+          <Link to="#file-picker">
+            <strong>EuiFilePicker</strong>
+          </Link>
+          .
+        </p>
+        <EuiCallOut
+          color="warning"
+          iconType="accessibility"
+          title={
+            <>
+              For accessibility, we recommend wrapping each from control in a{' '}
+              <strong>EuiFormRow</strong>
+            </>
+          }
+        >
+          <p>
+            Use the{' '}
+            <Link to="/forms/form-layouts#form-and-form-rows">
+              <strong>EuiFormRow</strong>
+            </Link>{' '}
+            component to easily associate form components with labels, help
+            text, and error text. It is highly recommended to provide a{' '}
```

Comment:
Here we'll mention accessbility and be more forceful about it

```suggestion
            text, and error text. For accessibility, you must provide a{' '}
```